manifests/h/Hashicorp/Nomad/1.7.3/Hashicorp.Nomad.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Hashicorp.Nomad
+PackageVersion: 1.7.3
+PackageLocale: en-US
+Publisher: HashiCorp
+PublisherUrl: https://www.hashicorp.com/
+PublisherSupportUrl: https://github.com/hashicorp/nomad/issues
+PrivacyUrl: https://www.hashicorp.com/privacy
+Author: David Lemler
+PackageName: Hashicorp Nomad
+PackageUrl: https://www.nomadproject.io/
+License: Mozilla Public License 2.0

```

```suggestion
License: Business Source License
```

https://github.com/hashicorp/nomad/commit/b3e30b1dfa185d9437a25830522da47b91f78816